plugins/rtmp-services/data/services.json

```diff
@@ -2808,6 +2808,75 @@
             "supported video codecs": [
                 "h264"
             ]
+        },
+        {
+            "name": "Livepeer Studio",
+            "more_info_link": "https://docs.livepeer.org/guides/developing/stream-via-obs",
+            "stream_key_link": "https://livepeer.studio/dashboard/streams",
+            "servers": [
+                {
+                    "name": "RTMP",
+                    "url": "rtmp://rtmp.livepeer.com/live"
+                }
+            ],
+            "recommended": {
+                "keyint": 1,
+                "profile": "high",
+                "bframes": 0,
+                "supported resolutions": [
```

Comment:
Can we omit this? I've streamed at 1440p before and I'd like to do so again. If we can't omit this, can we go up to 4K?

Reply:
Yeah, it makes sense to omit this, we have the global max bitrate